src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -542,41 +541,46 @@ public void deleteFeedbackSessionsForCourses(List<String> courseIds) {
     }
     
     @SuppressWarnings("unchecked")
-    private List<FeedbackSession> getFeedbackSessionEntitiesWithUnsentOpenEmail() {
+    private List<FeedbackSession> getFeedbackSessionEntitiesNeedingOpenEmail() {
         Query q = getPm().newQuery(FeedbackSession.class);
-        q.declareParameters("boolean sentParam, Enum notTypeParam");
-        q.setFilter("sentOpenEmail == sentParam && feedbackSessionType != notTypeParam");
+        q.declareParameters("java.util.Date startTimeParam, boolean sentParam");
+        q.setFilter("startTime > startTimeParam && sentOpenEmail == sentParam");
```

Comment:
`getSessionsPossiblyNeedingOpenEmail`? Add a header comment to explain which criteria is used exactly.